blobfuse-artifacts.yml

```diff
@@ -98,10 +98,6 @@ stages:
             DistroVer: "Debian9.0"
             AgentName : "DEB 9.0"
             Description: "Debian Linux 9.0"
-          Debian-10.0:
```

Comment:
We want to add ubn-22 to this pipeline ?

Reply:
That means we will start "supporting" Ubuntu 22. I'd hold off on that for now